### ISSUE TITLE
Show offline widget when operator went offline in the middle of chat

### ISSUE THREAD

**Issue by bharadwajag** [LiveHelperChat/livehelperchat#1078]
Hi,

I've configured an option to keep the conversation even though the chat widget is closed. 

My problem is, if a user starts the conversation when an operator is online, if suddenly operator went offline for some reason, I'm still able to see the conversation even though there are no online operators.

What I need is, either it should show the offline form to the user (or) should allow the user to send an email to the operator.

Can anyone please help me in this?

**Comment by remdex** [comment]
I do not think that this scenario is supported at the moment. If chat is started the only option is for operator to redirect visitor to contact form manually.
